src/service.ts

```diff
@@ -0,0 +1,13 @@
+import { Client } from "./client";
+
+export class Service {
+    
+    client: Client;
+
+    /**
```

Comment:
this JSDoc would show up for consumers, but not add any useful information.
Personally I'd much rather have no JSDoc than one that gives me no information.